instrumentation/opentelemetry-instrumentation-dbapi/tests/test_dbapi_integration.py

```diff
@@ -50,7 +50,7 @@ def test_span_succeeded(self):
         spans_list = self.memory_exporter.get_finished_spans()
```

Comment:
Would be nice to add a test for formats mentioned above like leading space or newline separators

Reply:
Done